.travis.yml

```diff
@@ -29,97 +59,101 @@ matrix:
           - UACLIENT_BEHAVE_BUILD_PR=${BUILD_PR} make test
     - if: env(UACLIENT_BEHAVE_AWS_SECRET_ACCESS_KEY)
       python: 3.7
-      env: TOXENV=behave-pro-aws
+      env: TOXENV=behave-aws-pro
       script:
-          - BUILD_PR=1
-          - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then BUILD_PR=0; fi
           - >
-              if [ "$BUILD_PR" -eq "1" ]; then
+              if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+                  BUILD_PR=0
+              else
                  cd $TRAVIS_BUILD_DIR/..
                  tar -zcf pr_source.tar.gz ubuntu-advantage-client
                  cp pr_source.tar.gz /tmp
                  ls -lh /tmp
                  cd $TRAVIS_BUILD_DIR
               fi
           - UACLIENT_BEHAVE_BUILD_PR=${BUILD_PR} make test
-    - python: 3.7
+    - if: env(UACLIENT_BEHAVE_CONTRACT_TOKEN)
+      python: 3.7
       env: TOXENV=behave-14.04
       script:
-          # bionic has lxd from deb installed, remove it first to avoid
-          # confusion over versions
-          - BUILD_PR=1
-          - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then BUILD_PR=0; fi
           - >
-              if [ "$BUILD_PR" -eq "1" ]; then
+              if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+                  BUILD_PR=0
+              else
                  cd $TRAVIS_BUILD_DIR/..
                  tar -zcf pr_source.tar.gz ubuntu-advantage-client
                  cp pr_source.tar.gz /tmp
                  ls -lh /tmp
                  cd $TRAVIS_BUILD_DIR
               fi
+          # bionic has lxd from deb installed, remove it first to avoid
```

Comment:
I think we should update this comment, since it stating that this is bionic related, but we are doing that on all of the releases